bmecat12/writer.go

```diff
@@ -40,6 +40,13 @@ type CatalogWriter interface {
 	Articles(context.Context) (<-chan *Article, <-chan error)
 }
 
+// Optionally implement this interface on your CatalogWriter
+// if you want to write a CATALOG_GROUP_SYSTEM.
+type CatalogGroupSystemWriter interface {
```

Comment:
The name of the comment should start with the interface name, the method name etc. Something like:

```
// CatalogGroupSystemWriter, when implemented by a consumer, allows to write
// a CATALOG_GROUP_SYSTEM.
type CatalogGroupSystemWriter interface {
  ...
}
```